sbin/travis.sh

```diff
@@ -79,6 +113,41 @@ RunTests() {
             sh -e /etc/init.d/xvfb start
         fi
     fi
+
+	# Install dependencies through Package Control
+	if [ -n $PCDEPS ]; then
+		STIP="${STP%/*}/Installed Packages"
+		if [ ! -d "$STIP" ]; then
+			echo creating sublime installed package directory
+			mkdir -p "$STIP"
+		fi
+	
+		# Install PackageControl
+		PC_URL="https://packagecontrol.io/Package Control.sublime-package"
+		PC_PKG="${PC_URL##*/}"
+		curl "$PC_URL" -o "$STIP/$PC_PKG"
+
+		# Cycle ST to complete installation
+		finished="! awk '/in_process_packages/,/]/' \
+			'$STP/User/Package Control.sublime-settings' | tail -n +2 | grep -q \\\""
+
+		echo Installing Package Control...
+		CycleUntil "[ -f '$STP/User/Package Control.sublime-settings' ] && $finished"
+
+		echo Opening Sublime until Package Control installs...
+		CycleUntil "[ -d '$STP/bz2' ] && $finished"
```

Comment:
Yes, you're right. Once PC gets SSL and whatever else installed, it will check and install every package's deps at startup. I had always intended for that extra call to be overkill, so now I've removed it.